Tools/Scripts/libraries/webkitscmpy/webkitscmpy/contributor.py

```diff
@@ -176,6 +176,8 @@ def create(self, name=None, *emails, **kwargs):
         def __iter__(self):
             yielded = set()
             for contributor in self.values():
+                if not contributor:
+                    continue
                 if contributor and contributor.name in yielded:
```

Comment:
We can stop checking `contributor` here.

Reply:
Good catch